Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -227,18 +225,11 @@ public class ManifestLoader: ManifestLoading {
             at: path
         ) + ["--tuist-dump"]
 
-        let result = System.shared
-            .observable(arguments, verbose: false, environment: environment.manifestLoadingVariables)
-            .toBlocking()
-            .materialize()
+        do {
+            let string = try System.shared.capture(arguments, verbose: false, environment: environment.manifestLoadingVariables)
```

Comment:
Worth knowing if there was any reason why before we were using the other method. Is `capture` getting only the standard output as well? or both standard and error output? could you double-check? 😄

Reply:
Capture does get the error output and passes it back (as an exception) if the process fails.

There are tests (`test_load_invalidFormat` and `test_load_missingManifest`) which cover this.

On a related note, it seems like 2 of the 4 `ManifestLoaderError` cases don't actually get used. In the case of the code being changed here, the error from the underlying system call is simply rethrown. It seems like `ManifestLoaderError.unexpectedOutput` might be more appropriate, but using that would display less information. Perhaps just remove the unused cases?